src/pages/profile/components/Review/index.tsx

```diff
@@ -0,0 +1,25 @@
+import ReviewContent from './ReviewContent';
+
+interface Review {
+  score: number;
+  content: string;
+}
+interface ReviewProps {
+  reviews: Review[];
+}
+const Review = ({ reviews }: ReviewProps) => {
+  return (
+    <div>
+      <span className='font-bold'>매너 리뷰</span>
+      <div className='py-1'>
+        {reviews.map(review => {
+          const { score, content } = review;
+
+          return <ReviewContent reviewNumber={score} reviewTitle={content} />;
+        })}
```

Comment:
아래처럼 좀 더 간결하게 작성할 수 있을 것 같아요~

```suggestion
        {reviews.map(({ score, content }) => (
          <ReviewContent reviewNumber={score} reviewTitle={content} />
        ))}
```